src/System.Security.Cryptography.Xml/src/Resources/SR.Designer.cs

```diff
@@ -0,0 +1,838 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
Ah, sorry, did not realise this. The problem seems to be that this project is currently excluded from the command-line build. So while I can make this change I will no longer be able to test it. Are you still ok with me making it at this stage?